src/cf-typography/usage.md

```diff
@@ -45,6 +45,12 @@ project by duplicating the variable `@key: value`.
 Color variables referenced in comments are from [cf-core cf-brand-colors.less](https://github.com/cfpb/capital-framework/blob/master/src/cf-core/src/cf-brand-colors.less).
 
 ```
+// Font variables
```

Comment:
Oh one change, you might move this to its own section so it's not hidden inside "Color variables".

Reply:
Ditto this then 👍